src/nodes/IfStatement.js

```diff
@@ -1,38 +1,58 @@
 /* eslint-disable implicit-arrow-linebreak */
```

Comment:
we shouldn't have these lines anymore

Reply:
I was waiting for #157 to be merged. now I rebased to master.